Packs/CommonScripts/Scripts/StixCreator/StixCreator.py

```diff
@@ -148,6 +148,13 @@ def main():
                     except KeyError:
                         pass
 
+                elif indicator_type == 'malware':
+
+                    if all_args[indicator_fields].get('ismalwarefamily', '').lower() == 'true':
+                        kwargs['is_family'] = True
+                    else:
+                        kwargs['is_family'] = False
```

Comment:
```suggestion
                    kwargs['is_family'] = argToBoolean(all_args[indicator_fields].get('ismalwarefamily', '').lower())
```

Reply:
yes, couldnt remember the helper function for it, thank you!